keyboards/ropro/keymaps/default/default.c

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2015-2017 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+extern keymap_config;
+
+#define _BASE 0
```

Comment:
Not required, but we've begun recommending use of `enum` to manage layer indexes.

Instead of https://github.com/qmk/qmk_firmware/blob/712deb9bffeee797da2b97860ac08a41b06d8df2/keyboards/ropro/keymaps/default/default.c#L21-L22

use:

```c
enum layer_names {
    _BASE,
    _LOWER
};
```

This is programmatically equivalent to what you have here, but doesn't require changing a bunch of defines if the number or order of layers changes.